### PR TITLE
Fixed flaky unit tests

### PR DESCRIPTION
**Pull request by mfaizanse**
**Description**

Changes proposed in this pull request:

- Fixed flaky unit tests

image

**Related issue(s)**

### CODE REVIEW

**Comment by mfaizanse** [comment]
pull-eventing-manager-unit-test

File: internal/controller/eventing/controller.go [hunk 1]
```diff
@@ -214,9 +214,6 @@ func (r *Reconciler) reconcileNATSBackend(ctx context.Context, eventing *eventin
 
 	// set NATSAvailable condition to true and update status
 	eventing.Status.SetNATSAvailableConditionToTrue()
-	if err := r.syncEventingStatus(ctx, eventing, log); err != nil {
-		return ctrl.Result{}, err
-	}
```
Comment (muralov) [inline internal/controller/eventing/controller.go:217]: What you deleted reflects the NATS availability status immediately. Let's says the condition is not stored in ETCD and EPP deployment is still in-progress, then the condition still displays NATS unavailable (although available).
Hence, I would leave the logic.

File: test/utils/integration/integration.go [hunk 2]
```diff
@@ -60,7 +60,7 @@ const (
 	namespacePrefixLength    = 5
 	TwoMinTimeOut            = 120 * time.Second
 	BigPollingInterval       = 3 * time.Second
-	BigTimeOut               = 30 * time.Second
+	BigTimeOut               = 60 * time.Second
```
Comment (muralov) [inline test/utils/integration/integration.go:63]: do you you really want to have 1 minute here. Maybe 40 seconds?

Comment (mfaizanse) [reply test/utils/integration/integration.go:63]: I have tested it 8 times with 1 minute and its seems stable, so lets keep this 1 minute.

**Review by muralov — APPROVED** [review]

**Pull request merged**